python/gLaSDI/examples/MFEMex9/README.md

```diff
@@ -0,0 +1,33 @@
+The gLaSDI framework is applied to construct a non-intrusive reduced-order model of the [**MFEM Example 9 Time-Dependent Radial Advection**](https://github.com/mfem/mfem/blob/master/examples/ex9.cpp).
+
+
+The data can be generated by `generate_data.ipynb`. Alternatively, some datasets are avaiable in `/usr/workspace/he10/data/MFEMex9/`, which are used in the numerical examples in the paper. The datasets are generated from the parameter space constituted by the parameters of the initial condition, including **w1** and **w2**.
```

Comment:
This can be updated, since people will not be able to access the additional datasets.

Reply:
I will update it with more details for external usage.